api-reference/10 UI Widgets/dxDataGrid/1 Configuration/export/export.md

```diff
@@ -310,7 +314,7 @@ The following instructions show how to enable and configure client-side export:
         })
         export class AppComponent {
             onExporting(e) {
-                const workbook = new ExcelJS.Workbook();    
+                const workbook = new Workbook();    
```

Comment:
Line 329: Angular won't compile until I specify the type for `buffer` as `BlobPart`

> workbook.xlsx.writeBuffer().then(function(buffer: BlobPart) {

Reply:
This one is still waiting to be fixed.